pkg/fakes/mock_eth_client.go

```diff
@@ -137,6 +120,14 @@ func (client *MockEthClient) SetTransactionSenderErr(err error) {
 	client.transactionSenderErr = err
 }
 
+func (client *MockEthClient) SetBalanceAtErr(err error) {
+	client.balanceAtErr = err
+}
+
+func (client *MockEthClient) SetBalanceAt(balance *big.Int) {
+	client.passedBalance = balance
```

Comment:
maybe this should be called `balanceToReturn` or something? `passedBalance` makes me think it's an argument received by one of the functions